kokkos/test/views.py

```diff
@@ -140,7 +140,10 @@ def test_view_access(self):
             self.assertEqual(_data[0].create_mirror_view()[_zeros], 0)
             self.assertEqual(_data[1].create_mirror_view()[_zeros], 0)
             self.assertEqual(_data[0].create_mirror_view()[_idx], 1)
-            self.assertEqual(_data[1].create_mirror_view()[_idx], 2)
+            if _kwargs["dtype"] == kokkos.bool:
+                self.assertEqual(_data[1].create_mirror_view()[_idx], True)
+            else:
+                self.assertEqual(_data[1].create_mirror_view()[_idx], 2)
```

Comment:
```suggestion
            value = lambda v : v if _kwargs["dtype"] != kokkos.bool else v > 0
            self.assertEqual(_data[1].create_mirror_view()[_idx], value(2))
```

Thus thought this scheme would be easier to maintain + more compact